tests/unit/baremetalpxeserver/test_pxeserver_handlers.py

```
@@ -0,0 +1,713 @@
+
+
+# ---------------------------------------------------------------------------
+# Import the baremetalpxeserver module with octal fix
+# ---------------------------------------------------------------------------
+try:
+    import tests.conftest  # noqa: F401
+
+    from tests.conftest import passthrough_lock
+    lock_mod = cast(object, importlib.import_module("zstacklib.utils.lock"))
+
+    setattr(lock_mod, "lock", passthrough_lock)
```

Ruff (0.15.2)

[warning] Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)